modules/cardano-explorer.nix

```diff
@@ -14,9 +14,7 @@ with (import ./../lib.nix);
       autoStart = true;
     };
 
-    networking.firewall.allowedTCPPorts = [
```

Comment:
We need firewall here enabled as well, let's revert this change. Explorer is also a cardano-node.